core/src/main/java/org/apache/iceberg/BaseTableScan.java

```diff
@@ -208,15 +205,14 @@ public InclusiveManifestEvaluator load(Integer specId) {
     int lookback = ops.current().propertyAsInt(
         TableProperties.SPLIT_LOOKBACK, TableProperties.SPLIT_LOOKBACK_DEFAULT);
     long openFileCost = ops.current().propertyAsLong(
-      TableProperties.SPLIT_OPEN_FILE_COST, TableProperties.SPLIT_OPEN_FILE_COST_DEFAULT);
+        TableProperties.SPLIT_OPEN_FILE_COST, TableProperties.SPLIT_OPEN_FILE_COST_DEFAULT);
 
     Function<FileScanTask, Long> weightFunc = file -> Math.max(file.length(), openFileCost);
 
     CloseableIterable<FileScanTask> splitFiles = splitFiles(splitSize);
-    return CloseableIterable.transform(
-        CloseableIterable.combine(
-            new BinPacking.PackingIterable<>(splitFiles, splitSize, lookback, weightFunc, true),
-            splitFiles),
+    return CloseableIterable.transform(CloseableIterable.combine(
```

Comment:
Was this change required? I liked that it was clear that all 3 lines were not passed to the same function. (Minor)